app/client/cli/event.go

```diff
@@ -157,8 +156,8 @@ func (c *Client) CmdEvents() *cobra.Command {
 			}
 
 			f := formatter{
-				header: []string{"EVENT TAG", "NAME", "# TEAM", "# EXERCISES", "CAPACITY"},
-				fields: []string{"Tag", "Name", "TeamCount", "ExerciseCount", "Capacity"},
+				header: []string{"EVENT TAG", "NAME", "# TEAM", "# EXERCISES", "CAPACITY", "CREATION TIME"},
```

Comment:
It is really messy when isolated features and bug fixes are handled in a non-version branch.

Reply:
Although I did not distinguish them properly, I agree.